src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityWrappers.cs

```diff
@@ -113,8 +113,6 @@ private static void InitializeAccessibilityDelegateAccessibleInterface()
         private static ulong AccessibilityCalculateStatesWrapper(IntPtr self, ulong initialStates)
         {
             View view = GetViewFromRefObject(self);
-            if (view == null)
```

Comment:
If view is null, then the following code is not accessible?
Or view cannot be null?
```
bitMask = view.AccessibilityCalculateStates().BitMask;
```

Reply:
It cannot be null, see below.

Reply:
Could you please throw exception than? If so, it might be helpful when some unknown error occured in future.

Reply:
Okay, added throwing `ArgumentException` in both `GetViewFromRefObject` and `GetInterfaceFromRefObject`.